### DISCUSSION TITLE
Connect Serverpod on server with Serverpod Insight?

### DISCUSSION THREAD

**Discussion by sulthanalihsan** [serverpod/serverpod#644] [Q&A] [answered]
thankyou for creating this excellent package!

I’ve successfully run my serverpod application locally and i can monitor it using serverpod insight, when I deploy to my server/vps can i or how can I connect my serverpod app on server with serverpod insight? thankyou

**Comment by yahu1031** [comment, marked as answer]
Once the deployment was done, open the insights app and left the bottom side you will see the environment to choose. And select production there and you are good to go.

**Reply by vlidholt** [reply]
Another tip is that if you are using a custom configuration (not the default AWS setup) you may need to edit the `config/production.yaml` file as it contains the information about how Insights will connect to your server.

**Reply by sulthanalihsan** [reply]
Great!. thankyou for your explanation @vlidholt @yahu1031

**Reply by innocent-greats** [reply]
@vlidholt May you help. I have configure the config file to 
<img width="375" alt="serverpod insights error" src="https://github.com/serverpod/serverpod/assets/97280792/b999e045-4530-421e-bfa9-0ddfb00bf31a">
the staging subdomain I have dedicated for insights using nginx and and when I test the url in the browser it works it gives me OK 2023-07-26 15:48:01.080695Z, however the insight app its giving me the server client exception 403 but the host url is the correct one I did set in config file

**Reply by KristijanZic** [reply]
Have you found a solution?

**Reply by yahu1031** [reply]
forbidden? Is this something we need to make changes in the hosting domain? @vlidholt
